test/simple_pre1.c

```diff
@@ -2,16 +2,16 @@
 #include <stdlib.h>
 #include "smack.h"
 
-int incr(int x) {
-  return x;
+short incr(short x) {
```

Comment:
Why did you change this test instead of creating another one with short?
Even more importantly, you cannot change simple_pre1.c without also updating simple_pre1_fail.c.
So please revert this test to what it used to be. And if you think that a similar test with shorts instead of ints would be useful, then create an additional test. I guess simple_pre4.c and simple_pre4_fail.c